install_cli.php

```diff
@@ -647,6 +663,49 @@ class FakeLogger { public function __call($m, $a) { } }
 		DataTool::$team_sets_array = $team_sets;
 	}
 
+	if ($module == 'ACLRoles' && !empty($_SESSION['tba'])) {
+		$tbaAccess = 72; //This value related to ACL_ALLOW_SELECTED_TEAMS(Owner & Selected Teams) constant
+		$roleActions = array(
+			'Accounts' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Contacts' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Leads' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Quotes' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Opportunities' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Bugs' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'Cases' => array('delete' => $tbaAccess, 'export' => $tbaAccess),
+			'KBContents' => array('delete' => $tbaAccess, 'export' => $tbaAccess)
+		);
+
+		$ACLRolesData = array();
+		$result = $GLOBALS['db']->query("SELECT id FROM acl_roles WHERE id LIKE 'seed-ACLRoles%'");
+		while($row = $GLOBALS['db']->fetchByAssoc($result)){
+			$ACLRolesData[$row['id']] = $row['id'];
+			$ACLRole = BeanFactory::retrieveBean('ACLRoles', $row['id']);
+			foreach ($roleActions as $category => $actions) {
+				foreach ($actions as $name => $access_override) {
+					$queryACL = "SELECT id FROM acl_actions where category='$category' and name='$name'";
+					$resultACL = $GLOBALS['db']->query($queryACL);
+					$actionId = $GLOBALS['db']->fetchByAssoc($resultACL);
+					if (!empty($actionId['id'])) {
+						$ACLRole->setAction($ACLRole->id, $actionId['id'], $access_override);

```

Comment:
This operation is also slow: would be much faster to write a separate `setAction` method that does the same, but does it for chunks of data and not one by one. But the pitfall of this approach is that if the behavior of `setAction` will be changed in core we will need to update the logic here.